si-timers/si-timers.cabal

```diff
@@ -59,7 +59,7 @@ library
                        stm,
                        time              >=1.9.1 && <1.13,
 
-                       io-classes       ^>=1.0
+                       io-classes        >=1.0 && <1.2
```

Comment:
If this is the only change made to the package between `1.0.0.0` and `1.1.0.0`, would a revision of `1.0.0.0` be sufficient? Or is it the goal to release the packages in lockstep with the same version for each package?

Reply:
I do release `io-classes`, `io-sim`, `strict-stm`, `strict-mvar` and `si-timers` (but not `io-classes-mtl`) in lock step, but I am fine to be more relaxed with lower bounds when I am sure things will still work.